src/ng/filter/filters.js

```diff
@@ -331,11 +331,15 @@ function formatNumber(number, pattern, groupSep, decimalSep, fractionSize) {
   }
 }
 
-function padNumber(num, digits, trim) {
+function padNumber(num, digits, trim, negWrap) {
   var neg = '';
   if (num < 0) {
-    neg =  '-';
-    num = -num;
+    if (negWrap) {
+      num = -num + 1;
```

Comment:
I know, but it is weird, because `0000-01-01` will be `0 BC` and `-0001-01-01` will be `2 BC`. It's kind of a gray area, so I'm not too bothered.

I jsut realized: what if `G` is not included in the format string ?
Then `-0026-01-01` will be `January 1st, 27`, right ? :confused: